l10n_ro_dvi/models/res_company.py

```diff
@@ -37,7 +37,7 @@ def _l10n_ro_get_or_create_custom_duty_product(self):
 
     def _l10n_ro_get_or_create_customs_commission_product(self):
         self.ensure_one()
-        customs_commission_product = self.l10n_ro_property_customs_commision_product_id
+        customs_commission_product = self.l10n_ro_property_customs_commission_product_id
```

Comment:
Ar trebui si un migration cu rename fields, avand in vedere ca deja il folosim in productie.

Reply:
am adaugat migration